src/algorithms/fardetectors/CMakeLists.txt

```diff
@@ -12,3 +12,5 @@ plugin_glob_all(${PLUGIN_NAME})
 # Find dependencies
 plugin_add_dd4hep(${PLUGIN_NAME})
 plugin_add_event_model(${PLUGIN_NAME})
+target_include_directories(${PLUGIN_NAME}_library PUBLIC ${TORCH_INCLUDE_DIR})
```

Comment:
We should move this into jana_plugin.cmake similar to plugin_add_dd4hep for consistency.